### DISCUSSION TITLE
any chance to get a wheel-pkg for python 3.11 ???

### DISCUSSION THREAD

**Discussion by tainnok** [christoph2/pyxcp#125] [General]
as written in the heading, is there any chance to provide wheel-packages for python 3.11?
As 3.11 should provide a big performance-increase i am looking to step up to 3.11 ... but i am not able to install/build pyxcp on python 3.11 (windows 10).

Any idea when pyxcp will be ship python 3.11 whl-packages?

**Comment by christoph2** [comment]
Python 3.11 wheels for Windows are available from now on.

While pyXCP works on Linux (incl. Raspbian 32/64), building wheels with Github Actions is still an open issue.

**Comment by christoph2** [comment]
Manylinux / x86_64 wheelz are now available.
